src/ExpressReceiver.ts

```diff
@@ -108,7 +108,7 @@ export default class ExpressReceiver implements Receiver {
         this.logger.debug('stored response sent');
       }
     } catch (err) {
-      res.send(500);
+      res.status(500).send();
```

Comment:
Although the warning message suggests to use `res.sendStatus(status)`, I prefer your change. In this case, a Slack app doesn't need to send any data in response body.
* https://github.com/expressjs/express/blob/4.16.1/lib/response.js#L136
* https://github.com/expressjs/express/blob/4.16.1/lib/response.js#L354-L359